.github/await_deployment.py

```
@@ -15,6 +15,23 @@
+    # … earlier code …
+
+    # Second request: fetch latest deployment
+    response = requests.get(url, headers=headers)
+    if response.status_code != 200:
+        raise Exception(f"Failed to fetch latest deployment: {response.text}")
+    deployments = response.json()['deployments']
+
+    # … some lines in between …
+
+    # Third request: check deployment status
+    current_response = requests.get(current_url, headers=headers)
```

Ruff (0.12.2)

Undefined name `current_url`

(F821)